quinn-proto/src/packet.rs

```diff
@@ -268,7 +350,7 @@ impl PacketNumber {
     }
 }
 
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, PartialEq)]
```

Comment:
We don't need it... so it would just be code size bloat?

Reply:
I don't think there's a meaningful code size cost; `Eq` is effectively a marker trait like `Send` and `Sync`, so usually if it's not implemented for an otherwise-comparable type there's a reason. Not very important, but might briefly confuse a reader.

Reply:
Ah, you're right. I've added `Eq`.